ibis/backends/tests/test_aggregation.py

```diff
@@ -466,6 +466,11 @@ def mean_and_std(v):
             lambda t, where: t.string_col.approx_nunique(where=where),
             lambda t, where: t.string_col[where].nunique(),
             id="approx_nunique",
+            marks=pytest.mark.xfail_version(
+                duckdb=["duckdb>=1.1"],
+                raises=AssertionError,
+                reason="not exact, even at this tiny scale",
+            ),
```

Comment:
Don't need to handle it in this PR, but we should probably split any approx function tests out to test approximate results rather than just xfailing.

Reply:
Agree.